chapter4.adoc

```diff
@@ -46,57 +46,65 @@ The `ZERO` field must always be 0.
 The `TYPE` field determines the interpretation of the `MTTL2` entry. The
 `TYPE` field encoding is as follows:
 
-* 0000b - `1G_disallow` - read and write access is not allowed for the 1G
-range for this supervisor domain
-* 0001b - `1G_allow_r` - read access for the 1G range is allowed for the
-supervisor domain
-* 0011b - `1G_allow_rw` - read and write access for the 1G range is allowed
-for the supervisor domain
-* 0100b - `MTT_L1_DIR` - The 32M range is composed of 8192x4K pages
-* 0111b - `2M_PAGES` - The 32M range is composed of 16x2M pages
+* 0000b - `1G_disallow` - read, write or execute access is not allowed for the
+           1G range for the supervisor domain.
+* 0001b - `1G_allow_rx` - read and execute access for the 1G range is allowed
+          for the supervisor domain.
+* 0010b - `1G_allow_rw` - read and write access for the 1G range is allowed
+          for the supervisor domain
+* 0011b - `1G_allow_rwx` - read, write and execute access for the 1G range is
+          allowed for the supervisor domain
+* 0100b - `MTT_L1_DIR` - The 32M range is composed of 8192 x 4K pages
+* 0111b - `2M_PAGES` - The 32M range is composed of 16 x 2M pages
 * All other encodings are `reserved` and cause an access violation
 
 The `INFO` field depends on the `TYPE` field and is formatted as per the
 following table:
 
+[[Smmtt-rw-l2-encoding]]
 .`MTTL2` Entry Type for Smmtt
 [width="100%",cols="25%,75%",options="header",]
 |===
 |*`MTTL2` Entry `TYPE`* |*Description, `INFO` and `TYPE` field encoding*
 |`1G_disallow` a|
-_The 1G range of address is not allowed for the domain._
+_read, write or execute is not allowed to this 1G address range for the domain._
+The `INFO` field must be 0.
+When configuring 1G ranges, RDSM ensures that 16 `MTTL2` entries, each
```

Comment:
that 16 `MTTL2` entries, each corresponding to 64M of address space
-> that **32** `MTTL2` entries, each corresponding to **32M** of address space